docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md

```diff
@@ -301,11 +315,11 @@ myBinding.setDataAsync('Hello World!', function (asyncResult) { });
 
 In the example, the first parameter is the value to set on `myBinding`. Because this is a text binding, the value is a `string`. Different binding types accept different types of data.
```

Comment:
```suggestion
In this example, the first parameter is the value to set on `myBinding`. Because this is a text binding, the value is a `string`. Different binding types accept different types of data.
```